app/src/main/java/com/android/partagix/model/item/item.kt

```diff
@@ -0,0 +1 @@
+data class Item(val id: Long, val idCategory: Long)
```

Comment:
No Item name ?
And many other fields ?

Reply:
Yes, je vais add des trucs, l'idée c'était juste de mettre les basics et ensuite n'importe qui peut add tant que c'est fait aux 2 endroits